cypress/e2e/tables-table.cy.js

```diff
@@ -1,11 +1,15 @@
 let localUser
+let user1
```

Comment:
Maybe call it like that to make it more readable what this is about

```suggestion
let targetUserTransfer
```